server/api/models/roles.py

```diff
@@ -14,14 +14,15 @@ class MemberRole(models.Model):
     workspace_role_id = models.ForeignKey(WorkspaceRole, on_delete=models.CASCADE)
     member_id = models.ForeignKey(WorkspaceMember, on_delete=models.CASCADE)
 
-class Permission(models.Model):
+class MemberPermissions(models.Model):
     date_created = models.DateTimeField(auto_now_add=True)
     date_modified = models.DateTimeField(auto_now=True)
-    name = models.CharField(max_length=30)
-    description = models.TextField(blank=True)
+    workspace_id = models.ForeignKey(Workspace, on_delete=models.CASCADE, null=True)
```

Comment:
Why null=true? There should never be a situation where perms are not part of a workspace so why allow it in the db?

Reply:
Django wasn't liking it without it due to there already being cols set as null from the previous model. But `null=True` simply sets whether the field is allowed to be null in the database. `blank=True` is what controls whether the field is actually required or not in forms. Info [here](https://stackoverflow.com/a/8609425/23362045).